lib/server.js

```diff
@@ -417,12 +417,12 @@ class Server extends KarmaEventEmitter {
     }
 
     processWrapper.on('unhandledRejection', (error) => {
-      this.log.error(`UnhandledRejection: ${error.message || String(error)}`)
+      this.log.error(`UnhandledRejection: ${error.stack || error.message || String(error)}`)
       reportError(error)
     })
 
     processWrapper.on('uncaughtException', (error) => {
-      this.log.error(`UncaughtException:: ${error.message || String(error)}`)
+      this.log.error(`UncaughtException:: ${error.stack || error.message || String(error)}`)
```

Comment:
While at it, can you also remove extra ":" character?

Reply:
done